### ISSUE TITLE
Problem with draft uvfbk

### ISSUE THREAD

**Issue by bgirardot** [freesewing/site#272]
See [here](https://freesewing.org/drafts/uvfbk)

I am about 1.8m tall and weigh a good 100 kg so I am usually like XL sized. When I look at the comparison, it looks pretty off.

I have double checked the measurements i think are relevant, shoulder to wrist, neck to waist, waist to hip.

Any ideas?

**Comment by joostdecock** [comment]
Hi there, thanks for reaching out. I think there's a few things we could do to make this draft more balanced.

**Decrease across back**

The across back is typically not more than 4 cm less than the shoulder to shoulder measurement. Yours is 8 cm less, and I think that's part of the issue.
Increasing your across back to 46 or 47 cm will make the armholes cut less deep into the back, which will in turn influence the sleeve head.

**Shoulder slope**

Your shoulder slope is very high. It's worth double checking it. Anything above 7cm makes me suspicious that something went wrong while measuring. 

Double check it, keeping an eye on the instructions: https://freesewing.org/docs/measurements/shoulderslope

**Hips**

There's a sharp reduction from your waist circumference (122 cm) to your hip circumference (109cm).

Give that your chest and waist are similar, and your hips narrower, we need to smooth that out a bit. 

I'd suggest to decrease the ease in your wait to 10cm, then increase the hip ease to the point where the side seam is smooth. You can play around with it a bit by redrafting and tweaking.


Let me know how these changes work for you, I'll keep this open in the mean while.

**Comment by bgirardot** [comment]
Thank you very very much. I will double check those measurements and/or adjust to get a little more in line.

**Comment by bgirardot** [comment]
@joostdecock Still not sure what I am have so wrong. I did go back and check the other measurements you suggested and did revise some of them both because we got different measurements (knowing which side to err on) which did not seem to have a lot of impact on the pattern, and I bumped them up a few cm just to try and get the pattern right. 

This is the result of several trys at getting this to look better and using improved, but slightly adjusted measurements: https://freesewing.org/drafts/cexsm

**Comment by joostdecock** [comment]
Haha, you really won the lottery with that sexism (cexsm) draft :)

On a more serious note, this looks so much better. The side seams are smooth, the shoulder slope looks natural, I really think this looks very well.

But, you are probably worried about the differences with the comparison. So let's look at what's different:

**Armhole depth**

The armhole in your draft dips deeper than in the drafts we're comparing to. Here's the armhole depth formula, lifted straight from the brian body block:

```
$this->setValueIfUnset('armholeDepth', $this->v('shoulderSlope') / 2 + $model->m('bicepsCircumference') * $this->o('armholeDepthFactor'));
```

Note that by armholedepth, we mean from the center back of the neck, which is why the shoulder slope is taken into account, although this does not technically make the armhole deeper, it just shifts it lower.

As you can see the armhole depth is determined by the biceps circumference. Larger arms need a bigger armhole. No surprises there.

Your biceps circumference is 44cm. The standard models we compare against have a biceps circumference that ranges from  33.5cm to 35cm. So it should come as no surprise that your armhole is deeper. You need that space.

**Sleevehead looks different**

Once again, your sleeve is wider and your sleevecap a bit higher because you need that space. 

**Sleeves are shorter**

Your shoulder to wrist measurement is 67cm. For our standard models we compare against, that measurements varies between 67cm to 70cm. So your sleevelength should be on par witht the smaller sample size. Which it is, but it's just shifted up so that doesn't show that clearly.

**Body is shorter**

Your center back neck to waist is 47cm. For our standard models we compare against, that measurements varies between 48.9cm and 50.5cm (*) so you're only a bit shorter than those, but it looks a bit off because your draft is shifted upwards.

(*) These *standard* measurements come from a book, but I find them to be weird at times. I have 48cm for this measurement (smaller than all of them) and I'm really tall.

**Conclusion**

Unless you've somehow mis-measured your biceps circumference, I believe this draft will be good for you.

**Comment by bgirardot** [comment]
Hi @joostdecock thank you very much for the additional feedback. I did in fact go back and remeasure my biceps and found them to be smaller so adjusted that. No idea what my partner and I did that we got so some many of them wrong the first time. I guess just first time with no clue, but your advice helped us to understand what the more reasonable values were.

So with that, this is what I have finally ended up with and it looks pretty good:

https://freesewing.org/drafts/zescn

Now let us see what a first time sewer does with that :)

Thank you again for all the support.

cheers
